index.ts

```diff
@@ -32,7 +32,8 @@ export {
   ValidationResult
 } from "./lib/util/validationError"
 
-export { executePromisesSequentially, clearCache } from "./lib/util/utils"
+export { executePromisesSequentially} from "./lib/util/utils"
```

Comment:
why do we need to export this ?

Reply:
Because we run this function in `azure-rest-api-specs` CI. Sadly, directly from `node.js`.

Reply:
I mean `clearCache`. I have no idea about `executePromiseSequentially`. As soon as we get rid of API calls from portal and `azure-restapi-specs` CIs, I can throw away all of these exports.

Reply:
not sure I follow this seems that it should be an internal function, only exported functions should be clear API calls like validate etc, but i saw now it is exported so I guess we have to stick with it. Not sure what you mean get rid of API calls, all node packages are in some sense libraries with APIs. sometimes they have a CLI wrapper, but thats irrelevant, at base they are libraries with an API